### ISSUE TITLE
problem for from constraint

### ISSUE THREAD

**Issue by sourir** [C2FO/nools#172]
i have rule like this:
          when {    n1: Number from [1,2,3,4,5];   }

but i got :
         [ Number, "n", "true" ] 
in js file.

Is it my fault?  
Thanks in advance!

**Comment by markbjerke** [comment]
No, what you are seeing is the default constraint which is, the boolean constant true.  It's important to remember that 'from' is providing a substitute for facts in working memory. It is providing facts that are not in working memory,  The from modifier is not a constraint, just a source of data.  So what you really have is: n1 : Number;  with a from modifier to provide the number fact(s).
